README.md

```diff
@@ -43,10 +43,11 @@ captsisko,bsisko@starfleet.com
 You may want to store multiple CSV files in a folder that's inside of Starfish, but not tracked by git. If you create a new folder named "CSVsToParse", it will not be tracked by git. Or, you can name the folder something else and add that folder name to the .gitignore file.
 
 #### Then, get yourself Github authentication credentials.
-Log in to GitHub and [register a new OAuth app](https://github.com/settings/applications/new) (you can find this under Profile > Settings > Developer Settings > OAuth Apps > "new OAuth App"). Starfish will not use the Homepage or Callback URL, but you will need to provide a valid URL in these fields in order to create the app. Fill out the form, click "Register Application", and obtain a Client ID and Client Secret.
+Log in to GitHub and [register a new personal access token](https://github.com/settings/tokens/new) (you can find this under Profile > Settings > Developer Settings > Personal access token > "Generate new token"). The default permissions should suffice, fill the "Note" field with e.g. "Starfish" or another description, and then click "Generate token". Copy the access token and store it as you will need it for the next step.
 
 #### Next, Create a file named .env, copy the contents of the .env.template file into it, and add your values to the new file.
-- Fill in the GitHub Client ID and Github Client Secret.
+- Fill in your GitHub user id in GITHUB_ID
```

Comment:
My understanding is that the user id isn't actually needed if you pass a personal token as the password.

Reply:
Hmm, as how I understand it, the personal access token is just an alternative to your normal password, thus your user id would be required. Beside that, since it's basic auth, I am not quite sure what to put for the username as alternative then.

Either way, I just followed the docs https://developer.github.com/v3/auth/#via-oauth-and-personal-access-tokens

Reply:
I just did a local test, and it seemed to work when I set my github id to be an empty string, and a personal access token as the password. If that works for you as well, the code and config could be simplified by leaving out the github id.

Reply:
Double checked it against an endpoint that requires authentication and it does seem to work. Applied changes for it.